### PR TITLE
Block TLS 1.0 and 1.1

### PR DESCRIPTION
**Pull request by scottbommarito**
https://github.com/nuget/engineering/issues/2617

### CODE REVIEW

**Comment by 304NotModified** [comment]
The link is dead? Or private?

**Comment by skofman1** [comment]
@304NotModified , it's private. We will announce TLS 1.0, 1.1 deprecation a few weeks prior to applying it. There's no concrete date yet.

File: tests/NuGetGallery.FunctionalTests.Core/NuGetGallery.FunctionalTests.Core.csproj [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>NuGetGallery.FunctionalTests</RootNamespace>
     <AssemblyName>NuGetGallery.FunctionalTests.Core</AssemblyName>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
```
Comment (joelverhagen) [inline tests/NuGetGallery.FunctionalTests.Core/NuGetGallery.FunctionalTests.Core.csproj]: Will this run on our CI? I recall a restriction to only use 4.6.2 right now.

Comment (scottbommarito) [reply tests/NuGetGallery.FunctionalTests.Core/NuGetGallery.FunctionalTests.Core.csproj]: Yup, I discovered that issue this morning. We probably just need to install the 4.7.2 SDK.

Comment (joelverhagen) [reply tests/NuGetGallery.FunctionalTests.Core/NuGetGallery.FunctionalTests.Core.csproj]: I think we should make this a concerted effort apart from TLS work. The rest of our projects are 4.6.2 so I don't see why this should move past our product code.

Comment (scottbommarito) [reply tests/NuGetGallery.FunctionalTests.Core/NuGetGallery.FunctionalTests.Core.csproj]: I've restarted the build machines and now it seems to work without this fix. Will revert.

File: tests/NuGetGallery.LoadTests/App.config [hunk 2]
```diff
@@ -1,65 +1,65 @@
...
-        <assemblyIdentity name="Microsoft.Web.XmlTransform" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-1.2.0.0" newVersion="1.2.0.0" />
+        <assemblyIdentity name="Microsoft.Web.XmlTransform" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
+        <bindingRedirect oldVersion="0.0.0.0-1.2.0.0" newVersion="1.2.0.0"/>
       </dependentAssembly>
       <dependentAssembly>
-        <assemblyIdentity name="EntityFramework" publicKeyToken="b77a5c561934e089" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-5.0.0.0" newVersion="5.0.0.0" />
+        <assemblyIdentity name="EntityFramework" publicKeyToken="b77a5c561934e089" culture="neutral"/>
+        <bindingRedirect oldVersion="0.0.0.0-5.0.0.0" newVersion="5.0.0.0"/>
```
Comment (agr) [inline tests/NuGetGallery.LoadTests/App.config]: why version 5 here?

**Comment by scottbommarito** [comment]
Turned out this was the incorrect way to do this. Closing in favor of https://nuget.visualstudio.com/NuGetMicrosoft/_git/NuGetDeployment/pullrequest/1068

**Pull request closed without merging**